index.js

```diff
@@ -1,16 +1,19 @@
+/* eslint-env node */
 const vueCompiler = require('vue-template-compiler');
 const vueNextCompiler = require('vue-template-es2015-compiler');
 const babelCore = require('babel-core');
+const findBabelConfig = require('find-babel-config');
 
 const transformBabel = src => {
+  const {config} = findBabelConfig.sync(process.cwd());
   const transformOptions = {
```

Comment:
yep，this can  be remove.  but considering that some people may not be configured.